committee/committee_test.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2019 IoTeX
```

Comment:
if we want to test private function voteFilter, you may do the following:
```
func TestVoteFilter(t *testing.T) {
  require := require.New(t)
  c := &Committee{
    voteThreshold: big.NewInt(12345),
  };
  // first xxx case to test
  vote1, err := types.NewVote(
      ...
   )
   require.NoError(err)
   require.False(committee.voteFilter(vote1))
   // second xxx case
   ...
```
Same for candidateFilter. You make it too complicated.